packages/lesmis/src/utils/useHomeUrl.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { useHistory, useRouteMatch } from 'react-router-dom';
+
+export const useHomeUrl = () => {
+  const { params } = useRouteMatch();
+  const { push } = useHistory();
+  const { locale } = params;
+  const homeUrl = `/${locale}`;
+
+  const isHomeUrl = path => path.replace(/\/$/, '') === homeUrl;
+
+  const pushToHomeUrl = push(homeUrl);
```

Comment:
This name makes me think we're pushing something onto the home url, rather than pushing the home url into history. What do you think about this
```suggestion
  const navigateToHomeUrl = push(homeUrl);
```